src/Entity/Mooc/AttachmentFile.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace AppBundle\Entity\Mooc;
+
+use AppBundle\EntityListener\Interfaces\EntityFileInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="mooc_attachment_file")
+ * @ORM\EntityListeners({"AppBundle\EntityListener\FileListener"})
+ */
+class AttachmentFile implements EntityFileInterface
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     *
+     * @Assert\NotBlank
+     */
+    private $title;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     */
+    private $path;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     */
+    private $extension;
+
+    /**
+     * @var UploadedFile|null
+     *
+     * @Assert\File
+     */
+    private $file;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getTitle(): ?string
+    {
+        return $this->title;
+    }
+
+    public function setTitle(string $title): void
+    {
+        $this->title = $title;
+    }
+
+    public function getPath(): ?string
+    {
+        return $this->path;
+    }
+
+    public function setPath(?string $path): void
+    {
+        $this->path = $path;
+    }
+
+    public function getFile(): ?UploadedFile
+    {
+        return $this->file;
+    }
+
+    public function setFile(?UploadedFile $file): void
+    {
+        $this->file = $file;
+    }
+
+    public function getPrefixPath(): string
+    {
+        return 'files/quizz';
```

Comment:
`/quizz` should not be here

Reply:
Where do you think it have to be ?

Reply:
OOOoooo no sorry , it's not only for quizz it's for videos too :D I'll changed by mooc ok ?